contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -832,20 +832,41 @@ contract DualAggregator is OCR2Abstract, OwnerIsCreator, AggregatorV2V3Interface
    * @notice median from the most recent report
    */
   function latestAnswer() public view virtual override returns (int256) {
-    return s_transmissions[s_hotVars.latestAggregatorRoundId].answer;
+    return s_transmissions[uint32(latestRound())].answer;
```

Comment:
i might be wrong here, but i believe that because `latestRound()` does computations, using it's return value as the lookup key here would incur some small amount of gas cost.

Reply:
Yes, but this extra computation is what we want. 

The latestRound() function dynamically calculates the latest available round, the one we'll need to use in all the call functions.